src/fuzzfetch/fetch.py

```diff
@@ -86,20 +85,25 @@ def _extract_file(zip_fp, info, path):
 
 
 class BuildFlags(collections.namedtuple('BuildFlagsBase', ('asan', 'debug', 'fuzzing', 'coverage'))):
-    "Flags used to make a build"
+    """Class for storing TaskCluster build flags"""
 
     def build_string(self):
-        """Taskcluster denotes builds in one of two formats - i.e. linux64-asan or linux64-asan-opt
+        """
+        Taskcluster denotes builds in one of two formats - i.e. linux64-asan or linux64-asan-opt
         The latter is generated. If it fails, the caller should try the former.
         """
-        return (('-fuzzing' if self.fuzzing else '') +
-                ('-asan' if self.asan else '') +
-                ('-ccov' if self.coverage else '') +
-                ('-debug' if self.debug else '-opt'))
+        yield (('-fuzzing' if self.fuzzing else '') +
+               ('-asan' if self.asan else '') +
+               ('-ccov' if self.coverage else '') +
+               ('-debug' if self.debug else '-opt'))
+        yield (('-fuzzing' if self.fuzzing else '') +
+               ('-asan' if self.asan else '') +
+               ('-ccov' if self.coverage else '') +
+               ('-debug' if self.debug else ''))
```

Comment:
This shouldn't be changed. `build_string()` returns a string now instead of generating both variants.